### DISCUSSION TITLE
NEXT_PUBLIC_VERCEL_PROJECT_PRODUCTION_URL problem when 2 production urls

### DISCUSSION THREAD

**Discussion by trompx** [vercel/community#7367] [Idea]
Hi,

I was using NEXT_PUBLIC_VERCEL_PROJECT_PRODUCTION_URL to set a global url usable across my website.
The problem is I just added a production domain, so now I have 2 productions urls:
- mydomain.com
- myproject.vercel.app

But now, when I visit the website with the myproject.vercel.app, which gets its global url from NEXT_PUBLIC_VERCEL_PROJECT_PRODUCTION_URL, it is set to mydomain.com as it is the **shortest production url**. So all my links are broken.

Can you please make the vercel.app url accessible under another env var, like a default one: NEXT_PUBLIC_VERCEL_DEFAULT_PROJECT_PRODUCTION_URL.

**Comment by pawlean** [comment]
Hi @trompx! 

Could you use another environment variable, perhaps? For example, `NEXT_PUBLIC_VERCEL_DEFAULT_PROJECT_PRODUCTION_URL`, and set its value to `myproject.vercel.app`?

[a comment by an automated account is not shown]
